### DISCUSSION TITLE
Isotope standards

### DISCUSSION THREAD

**Discussion by johanvonboer** [humlab-sead/sead_change_control#292] [General]
In tbl_isotopes there is a column "isotope_standard_id". This is always null.

However, in tbl_isotope_measurements there is also a column called "isotope_standard_id" and this is always 1.

Should perhaps isotope_standard_id in tbl_isotopes be removed? Or is there a valid reason to why both of these tables refer to an isotope standard?

**Comment by johanvonboer** [comment]
Mats Eriksson should know why we do not have any information about isotope standard 1.

**Comment by johanvonboer** [comment]
@MGE4

**Comment by MGE4** [comment]
Yes, there is a reason for Isotope_standard_ID, this is because there are multiple standard elements that a lab might use for calibrating there data, functionally informing the instrument and the data what is measured. We only have samples from one lab so there standard is what "Isotope_standard_ID = 1" is. We have no data on what that standard is but the lab should. 

There is a problem with standards, one is that the major world wide standard gas that is used for this, is only produced in a lab in the US and it is very expensive to bye for laboratories. This leads to other labs to making there own standard to calibrate by. Having a way of communication what standard and what batch of that standard was used in producing a dataset could be crucial in the future if one of these standards is found to be "off" in some way, this could invalidate a complete dataset.

Therefor, having a way to trace the standard used in producing a dataset is important.

**Comment by johanvonboer** [comment]
@MGE4 do you know what lab we have received the current isotope data from? There is nothing (except a placeholder) in tbl_dataset_masters, which is where I would expect to find this information:
![image](https://github.com/humlab-sead/sead_change_control/assets/34707290/d43badf9-6b00-4608-a9c5-7fab346e0c80)

**Comment by MGE4** [comment]
Oh yes, we received the data from "BioArCh" https://www.york.ac.uk/archaeology/research/research-centres/bioarch/
